src/index.ts

```diff
@@ -1,11 +1,36 @@
-import { configure as configureLogging } from "./logger";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { configure as configureLogging, getLogLevel } from "./logger";
 import { run as stackReport } from "./stack-report";
+import { LOG_LEVELS } from "./types";
 
-configureLogging();
+void yargs(hideBin(process.argv))
+  .command(
+    "overview",
+    "list stacks, contained resources, and if generated by @guardian/cdk",
+    (yargs) => {
+      yargs.option("prefer-cache", {
+        type: "boolean",
+        description: "use local cache and fail is not found",
+      });
+      yargs.option("log-level", {
+        type: "string",
+        description: `define log level (${LOG_LEVELS.join(",")})`,
```

Comment:
minor: does `yargs` have a concept of "allowed values" for this option?

Reply:
Looks like `choices` is possible 👍 so have added that. Thanks.